### PR TITLE
Editing-interview-blog

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/VD1DDQVN/3955-improve-interview-prep-blog-post-for-apply-candidates

### Context

The Apply team want to link to this blog, so we should remove the image as it's pretty low quality and edit the footer to link to advisers.

### CODE REVIEW

File: app/views/content/blog/11-top-tips-for-interview-success.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ keywords:
 tags:
   - becoming a teacher
   - applications
-closing_paragraph: enriching-the-lives-of-young-people
 ---
 
-$interview$
+Caroline Downing is the Primary Programme Leader at the [Bedfordshire Schools' Training Partnership](https://bedsscitt.org.uk). She interviews applicants and assesses their potential to become successful classroom professionals.
```
Comment (EllieNodder) [inline app/views/content/blog/11-top-tips-for-interview-success.md:22]: @gemmadallmandfe moved this info to the top - felt better situated here?

**Comment by EllieNodder** [comment]
@gemmadallmandfe do we remove the reference to COVID and face masks?

File: app/views/content/blog/11-top-tips-for-interview-success.md (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/content/blog/11-top-tips-for-interview-success.md:22]: Agree now that we have lost the image on this page

**Comment by gemmadallmandfe** [comment]
> @gemmadallmandfe do we remove the reference to COVID and face masks?

I think that's a good idea - could just remove `In times of COVID 19,` and `face mask` - the rest is still valid?

File: app/views/content/blog/11-top-tips-for-interview-success.md [hunk 2]
```diff
@@ -17,10 +17,11 @@ keywords:
...
+
+She believes the key to receiving an offer for a training place lies in effective interview preparation.
```
Comment (gemmadallmandfe) [inline app/views/content/blog/11-top-tips-for-interview-success.md:24]: @EllieNodder Is it clear that the numbered section below is written by Caroline? Would it be worth adding something like 'Here are her tips....' after this sentence?

Happy for you to disagree!

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**